torch/csrc/jit/python/script_init.cpp

```diff
@@ -846,6 +846,22 @@ void initJitScriptBindings(PyObject* module) {
     }
   }
 
+  struct DeepCopyMemoTable {
+    std::shared_ptr<IValue::HashAliasedIValueMap> map;
+  };
+  py::class_<DeepCopyMemoTable>(m, "DeepCopyMemoTable");
```

Comment:
@zdevito I couldn't figure out a better way to store the table in the map and get a mutable reference to it later via pybind so I added this indirection. (Issue was in `py::cast<HashedAliasedIValueMap>`, it claimed that the object was in the scope of the cast function or something)

Reply:
This seems fine.